src/explorer/AzureAccountTreeItem.ts

```diff
@@ -54,4 +40,47 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
         }
         return super.compareChildrenImpl(item1, item2);
     }
+
+    public async pickTreeItemImpl(expectedContextValues: (string | RegExp)[]): Promise<AzExtTreeItem | undefined> {
+        const subscription: string = localize('subscription', 'subscription');
+        const subscriptionOrTrialApp: string = localize('subscriptionOrTrialApp', 'subscription or trial app');
+        const trialAppOrSignIn: string = localize('trialAppOrSignIn', 'trial app, sign in, or create a free Azure account');
```

Comment:
Not a blocker, but I'm not sure about this last label.  Might be good to just say trial app.